### PR TITLE
Gammaray packet initialization documentation

### PR DESCRIPTION
**Pull request by CePowers**
### :pencil: Description

**Type:** : :memo: `documentation`

I created a new file for the gamma ray documentation within the physics walkthrough. I put information on packet initialization, including packet properties.

### :pushpin: Resources

Examples, notebooks, and links to useful references.

### :vertical_traffic_light: Testing

How did you test these changes?

- [ ] Testing pipeline
- [ ] Other method (describe)
- [ ] My changes can't be tested (explain why)

### :ballot_box_with_check: Checklist

- [ ] I requested two reviewers for this pull request
- [ ] I updated the documentation according to my changes
- [ ] I built the documentation by applying the `build_docs` label

> **Note:** If you are not allowed to perform any of these actions, ping (@) a contributor.

### CODE REVIEW

File: docs/physics/tardisgamma/gammaraypacketinitialization.rst [hunk 1]
```diff
@@ -0,0 +1,20 @@
+*********************
+Tardis :math:`\gamma`
+*********************
+
+Packet Initialization
+=====================
+
+The gamma ray portion of tardis also uses packets of photons, which are given the following properties:
```
Comment (Knights-Templars) [inline docs/physics/tardisgamma/gammaraypacketinitialization.rst:8]: Please add references!

File: docs/physics/tardisgamma/gammaraypacketinitialization.rst [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+- **Packet Energy:** This is the energy of the packet in the comoving frame and is equal to the total energy divided by the number of packets.
+- **Packet Frequency:** This is the frequency of the packet in the comoving frame and is equal to the packet energy divided by Planck's constant, h.
+- **Packet Direction:** This is the direction of the packet and is made up of two angles, :math:`\theta` and :math:`\phi`. 
+:math:`\theta` is a polar angle between 0 and :math:`\pi` and :math:`\phi` is an azimuth angle between 0 and 2:math:`\pi`.
```
Comment (Knights-Templars) [inline docs/physics/tardisgamma/gammaraypacketinitialization.rst:12]: Please remove the bullets.

File: docs/physics/tardisgamma/gammaraypacketinitialization.rst (hunk 1 above)
Comment (CePowers) [reply docs/physics/tardisgamma/gammaraypacketinitialization.rst]: I added the references.

File: docs/physics/tardisgamma/gammaraypacketinitialization.rst (hunk 2 above)
Comment (CePowers) [reply docs/physics/tardisgamma/gammaraypacketinitialization.rst]: I removed the bullets

**Pull request closed without merging**